src/Pages/LoginPage/Login.css

```diff
@@ -4,52 +4,44 @@
 }
 .loginBox{
     position: absolute;
-    width: 600px;
-    height: 600px;
-    left: 100px;
-    top: 240px;
-    border: 5px solid #F6F8EA;
-    box-sizing: border-box;
-    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
-    border-radius: 20px;
+    padding-left: 100px;
+    display: flex;
+    flex-direction: column;
+    
 }
 .email{
-    position: absolute;
     width: 200px;
     height: 33px;
     left: 20px;
-    top: 60px;
+    padding-top: 50px;
     color: white;
     font-style: normal;
     font-weight: bold;
     font-size: 24px;
 }
 .emailbtn{
```

Comment:
.emailBtn and .passwordbtn contains duplicated logic.
Also follow proper camel case in naming conventions